subjects/devops/concat-string/README.md

```diff
@@ -0,0 +1,29 @@
+## concat_string
+
+### Instructions
+
+Create a file `concat_string.py` that contains a function `concat` which takes in two strings as parameters and performs the following operations:
+
+- Concatenates the two strings by adding a comma and space between them.
+- Returns the resulting string.
+
+### Usage
+
+Here is an example of how to use the `concat` function:
```

Comment:
In other exercises, it has been used a slightly different approach. 

The python code is presented as the content of a script `test.py`. It is then reported the behaviour of the script as follows:
```console
$ python3 test.py
<expected output>
$
```

Reply:
Done. Thanks